src/app/page/Donate/DonationPage.jsx

```diff
@@ -170,7 +171,8 @@ function ErrorState({ setState }) {
       </Typography>
       <Typography align="left" variant="body1" gutterBottom>
         Otherwise, our volunteers can help you donate through a different method, such as by check.
-        For more information please call: <a href={`tel:${org.phoneNumber}`}>{org.phoneNumber}</a>
+        For more information please call:{" "}
```

Comment:
what does {" "} do ?

Reply:
eslint forces the jsx to the next line which removes the space that would naturally be there. this just manually adds that space back in